app/src/main/java/net/cassiolandim/florianopolisroutes/activity/DetailsActivity.java

```diff
@@ -0,0 +1,85 @@
+package net.cassiolandim.florianopolisroutes.activity;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.ActionBar;
+import android.support.v7.widget.Toolbar;
+
+import net.cassiolandim.florianopolisroutes.R;
+import net.cassiolandim.florianopolisroutes.fragment.DeparturesListFragment;
+import net.cassiolandim.florianopolisroutes.fragment.StopsListFragment;
+
+public class DetailsActivity extends ParentActivity {
+
+    public static final String EXTRA_ID = "EXTRA_ID";
+    public static final String EXTRAS_NAME = "EXTRAS_NAME";
+
+    private long routeId;
+
+    private ViewPager mViewPager;
+    private FragmentPagerAdapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        routeId = getIntent().getLongExtra(EXTRA_ID, 0);
+        if (routeId == 0) {
+            finish();
+            return;
+        }
+
+        String routeName = getIntent().getStringExtra(EXTRAS_NAME);
+
+        setContentView(R.layout.activity_details);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        ActionBar ab = getSupportActionBar();
+        ab.setTitle(routeName);
+        ab.setDisplayHomeAsUpEnabled(true);
```

Comment:
It would be nice to extract a method exclusive to deal with the action bar. The variable name, not so much, it's such a short scope, but wouldn't hurt =D